perfkitbenchmarker/benchmarks/cassandra_stress_benchmark.py

```diff
@@ -157,19 +191,31 @@ def RunTestOnLoader(vm, loader_index, keys_per_vm, data_node_ips):
     keys_per_vm: The number of keys per loader vm need to insert.
     data_node_ips: List of IP addresses for all data nodes.
   """
+  cassandra_stress_command = FLAGS.cassandra_stress_command
+
+  if FLAGS.cassandra_stress_command == USER_COMMAND:
+    cassandra_stress_command += ' profile={profile} ops\({ops}\)'.format(
+        profile=TEMP_PROFILE_PATH,
+        ops=FLAGS.cassandra_stress_ops)
+    schema_option = ''
+  else:
+    schema_option = '-schema replication\(factor={replication_factor}\)'.format(
```

Comment:
It would be nice if we could also specify the replication strategy in addition to the replication factor (so that it would do intelligent things if running in multiple zones), but to take advantage of that, the Cassandra configuration would have to be smarter (i.e. we need to fill out the cassandra-rackdc.properties file based off of VM zones).